### DISCUSSION TITLE
How can i move npm from my docker container to proxmox lxc container

### DISCUSSION THREAD

**Discussion by balajeek** [tteck/Proxmox#1370] [General]
I am new to proxmox, and been moving stuff from my ubuntu based docker containers, I have ngnix proxy manager on docker with 30+ proxy host defined some with let's encrypt certificates, how can i backup and restore into proxmox lxc?

on npm docker the volumne is mapped to local folder on ubuntu, where i see bunch of folders, files, mysql extra.
on proxmox lxc, i used the proxmox help script to install ngnix proxy manager, so not sure where npm is installed and the configuration files are and where msql is.

how can i move if it all possible?

**Comment by btehan** [comment]
Could this page be helpful?

https://github.com/NginxProxyManager/nginx-proxy-manager/discussions/1529
